tests/test.rs

```diff
@@ -32,8 +32,10 @@ fn make_keyed_auth(kp: &Keypair, msg: &external::Message) -> external::KeyedAuth
 struct Token(Env, [u8; 32]);
 
 impl Token {
-    fn initialize(&mut self, admin: &Identifier) {
-        external::initialize(&mut self.0, &self.1, admin);
+    fn initialize(&mut self, admin: &Identifier, decimal: u32, name: &str, symbol: &str) {
```

Comment:
Does it work to accept `&str` here? Swish.

Reply:
Not sure I understand. Are you referring to the `external::initialize` call on line 38?

Reply:
38 where we are using `&str` as params. I just haven't seen that work yet, so cool if it does!

Reply:
38 uses `stellar_contract_sdk::Vec` for the new params. `&str` doesn't work there, which is why I construct the Vec.